karyon3-admin-simple/src/main/java/com/netflix/karyon/admin/HttpServerRegistry.java

```diff
@@ -0,0 +1,11 @@
+package com.netflix.karyon.admin;
+
+import java.util.Map;
+
+import javax.inject.Provider;
+
+public interface HttpServerRegistry {
+    Map<String, Provider<SimpleHttpServer>> getServers();
```

Comment:
Why parallel maps rather than a single map with the value being a container for the provider and the config?